spec/mailers/authentication_mailer_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe AuthenticationMailer, type: :mailer do
+  subject(:mailer) { described_class }
+
+  describe 'send sign_up email' do
+    let(:token) { 'blub' }
+    let(:mail) { mailer.send_sign_up(to: 'test@example.com', token: token) }
+
+    before { mail.deliver! }
+
+    it 'can send a sign up email with the correct subject' do
```

Comment:
The `--format documentation` output for this is quite repetitive

```
AuthenticationMailer
  send sign_up email
    can send a sign up email with the correct body
    can send a sign up email with the correct heading
    can send a sign up email with a magic link
    can send a sign up email with the correct subject
```

This could be terser — maybe

```
AuthenticationMailer
  .send_sign_up_email
    sends an email with the correct body
    # etc
```